datadog/fwprovider/data_source_datadog_aws_cur_config.go

```diff
@@ -0,0 +1,176 @@
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/terraform-providers/terraform-provider-datadog/datadog/internal/utils"
+)
+
+var (
+	_ datasource.DataSource = &datadogAwsCurConfigDataSource{}
+)
+
+type datadogAwsCurConfigDataSource struct {
+	Api  *datadogV2.CloudCostManagementApi
+	Auth context.Context
+}
+
+type datadogAwsCurConfigDataSourceModel struct {
+	// Datasource ID
+	ID types.String `tfsdk:"id"`
+
+	// Query Parameters
+	CloudAccountId types.Int64 `tfsdk:"cloud_account_id"`
+
+	// Computed values
+	AccountId       types.String         `tfsdk:"account_id"`
+	BucketName      types.String         `tfsdk:"bucket_name"`
+	BucketRegion    types.String         `tfsdk:"bucket_region"`
+	CreatedAt       types.String         `tfsdk:"created_at"`
+	ReportName      types.String         `tfsdk:"report_name"`
+	ReportPrefix    types.String         `tfsdk:"report_prefix"`
+	Status          types.String         `tfsdk:"status"`
+	StatusUpdatedAt types.String         `tfsdk:"status_updated_at"`
+	UpdatedAt       types.String         `tfsdk:"updated_at"`
+	ErrorMessages   types.List           `tfsdk:"error_messages"`
+	AccountFilters  *accountFiltersModel `tfsdk:"account_filters"`
+}
+
+func NewDatadogAwsCurConfigDataSource() datasource.DataSource {
+	return &datadogAwsCurConfigDataSource{}
+}
+
+func (d *datadogAwsCurConfigDataSource) Configure(_ context.Context, request datasource.ConfigureRequest, response *datasource.ConfigureResponse) {
+	providerData, _ := request.ProviderData.(*FrameworkProvider)
+	d.Api = providerData.DatadogApiInstances.GetCloudCostManagementApiV2()
+	d.Auth = providerData.Auth
+}
+
+func (d *datadogAwsCurConfigDataSource) Metadata(_ context.Context, request datasource.MetadataRequest, response *datasource.MetadataResponse) {
+	response.TypeName = "aws_cur_config"
+}
+
+func (d *datadogAwsCurConfigDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, response *datasource.SchemaResponse) {
+	response.Schema = schema.Schema{
+		Description: "Use this data source to retrieve information about a specific Datadog AWS CUR (Cost and Usage Report) configuration. This allows you to fetch details about an existing Cloud Cost Management configuration for AWS billing data access.",
+		Attributes: map[string]schema.Attribute{
+			// Datasource ID
+			"id": utils.ResourceIDAttribute(),
+			// Query Parameters
+			"cloud_account_id": schema.Int64Attribute{
+				Required:    true,
+				Description: "The Datadog cloud account ID for the AWS CUR configuration you want to retrieve information about.",
+			},
+			// Computed values
+			"account_id": schema.StringAttribute{
+				Computed:    true,
+				Description: "The AWS account ID associated with this CUR configuration.",
+			},
+			"bucket_name": schema.StringAttribute{
+				Computed:    true,
+				Description: "The S3 bucket name where Cost and Usage Report files are stored.",
```

Comment:
Not sure it matters too much but sometimes you use `CUR` and other times it says `Cost and Usage Report`